metricbeat/module/mongodb/status/_meta/fields.yml

```diff
@@ -4,10 +4,12 @@
     MongoDB server status metrics.
   release: ga
   fields:
+    # Service.version?
     - name: version
       type: keyword
       description: >
         Instance version.
+    # Map to process.*?
```

Comment:
Not sure if we should map this as that the process name is mongod should be kind of obvious. @webmat Thoughts?

Reply:
Follow up PR here: https://github.com/elastic/beats/pull/10368